tests/system/matcher/eventually.py

```diff
@@ -0,0 +1,55 @@
+import common
+import retrying
+from precisely import Matcher
+from precisely.results import matched, unmatched
+
+
+class AppFailureMessage(Matcher):
+
+    def __init__(self, error_message):
+        self.expected = error_message
+
+    def match(self, actual):
+        message = actual['lastTaskFailure']['message']
+        # TODO: verify that item is app
+        if self.expected in message:
+            return matched()
+        else:
+            return unmatched("failure message '{}'".format(self.expected))
+
+    def describe(self):
+        return "failure message '{}'".format(self.expected)
+
+
+class HasEventually(Matcher):
+
+    def __init__(self, matcher):
+        self.matcher = matcher
+
+    def match(self, item):
+        assert callable(item), "The actual value is not callable."
+
+        @retrying.retry(
```

Comment:
We need to somehow parametrize `HasEventually` e.g. like ScalaTest's `patienceConfig`. Otherwise, this here only waits for 3s which is not applicable in most tests.

Reply:
@zen-dog, could you take another look?